src/qt/modaloverlay.h

```diff
@@ -9,7 +9,7 @@
 #include <QWidget>
 
 //! The required delta of headers to the estimated number of available headers until we show the IBD progress
-static const int REQ_HEADER_HEIGHT_DELTA_SYNC = 24;
+static constexpr int HEADER_HEIGHT_DELTA_SYNC = 24;
```

Comment:
No big deal since it's just one more line diff, but out of curiosity, why rename the constant?